servant-minimalist.hsfiles

```diff
@@ -0,0 +1,232 @@
+{-# START_FILE package.yaml #-}
+name:                {{name}}
+version:             0.1.0.0
+github:              "{{github-username}}{{^github-username}}githubuser{{/github-username}}/{{name}}"
+license:             BSD3
+author:              "{{author-name}}{{^author-name}}Author name here{{/author-name}}"
+maintainer:          "{{author-email}}{{^author-email}}example@example.com{{/author-email}}"
+copyright:           "{{copyright}}{{^copyright}}{{year}}{{^year}}2019{{/year}} {{author-name}}{{^author-name}}Author name here{{/author-name}}{{/copyright}}"
+
+extra-source-files:
+  - README.md
+  - ChangeLog.md
+
+# Metadata used when publishing your package
+# synopsis:            Short description of your package
+# category:            {{category}}{{^category}}Web{{/category}}
+
+# To avoid duplicated efforts in documentation and dealing with the
+# complications of embedding Haddock markup inside cabal files, it is
+# common to point users to the README.md file.
+description:         Please see the README on GitHub at <https://github.com/{{github-username}}{{^github-username}}githubuser{{/github-username}}/{{name}}#readme>
```

Comment:
This description is bad. The description should describe, not to link forward.

Reply:
This description comes from the old Servant template. I will try to write an explicit description. What - in your opinion - should appear here? Since it is at the user's discretion, something as simple as `<Fill me>`?

Reply:
I agree with HVR's view on `synopsis`, `description` and `README.md` use cases: https://mail.haskell.org/pipermail/haskell-cafe/2016-November/125541.html

I'd like the placeholders for `synopsis` and `description` to reflect that.